### DISCUSSION TITLE
Locking whisper model version

### DISCUSSION THREAD

**Discussion by AndyLogi** [openai/whisper#1734] [Q&A] [answered]
I would like to lock the version of the base.en model so that if improvements are made in the future I'm still using the same model.
I can see that the model is downloaded from a specific URL, stored in the `_MODELS` dictionary.  

If a new model is created, will:
1) the model at that URL be replaced; or
2) a new PyPI version will be released with an updated dictionary.

Essentially, my question boils down to if I lock to `openai-whisper==20230314` will I always be using the same `base.en` model or is there potential that this will change in the future?

**Comment by jongwook** [comment, marked as answer]
Yes! the code even has a SHA1 check to make sure it exactly matches with the intended checkpoint.

https://github.com/openai/whisper/blob/b38a1f20f4b23f3f3099af2c3e0ca95627276ddf/whisper/__init__.py#L20
